dask/dataframe/core.py

```diff
@@ -11,6 +11,7 @@
 
 from toolz import merge, partial, first, partition, unique
 import pandas as pd
+import bcolz
```

Comment:
This will need to be guarded with a `try-except` for `ImportError`s.  We can't assume that all users have bcolz installed.  It'd also be nice to isolate core.py from this if possible.

Looking more deeply now it looks like this isn't used in core and so can probably be removed.